modules/govuk_env_sync/files/govuk_env_sync.sh

```diff
@@ -206,18 +206,28 @@ function  dump_postgresql {
 }
 
 function restore_postgresql {
+  # Check which postgres instance the database needs to restore into
+  # (warehouse, transition, or postgresql-primary).
+  if [ "${database}" == 'transition_production' ]; then
+    db_hostname='transition-postgresql-primary'
+  elif [ "${database}" == 'content_performance_manager_production' ]; then
+    db_hostname='warehouse-postgresql-primary'
+  else
+    db_hostname='postgresql-primary'
+  fi
+
 # Checking if the database already exist
 # If it does we will drop the database
   DB_OWNER=''
-  if sudo psql -U aws_db_admin -h postgresql-primary --no-password --list --quiet --tuples-only | awk '{print $1}' | grep -v "|" | grep -qw "${database}"; then
+  if sudo psql -U aws_db_admin -h ${db_hostname} --no-password --list --quiet --tuples-only | awk '{print $1}' | grep -v "|" | grep -qw "${database}"; then
```

Comment:
Sorry to add tedium, but the quoting has changed in the meantime... 
```if sudo psql -U aws_db_admin -h "${db_hostname}" --no-password --list --quiet --tuples-only | awk '{print $1}' | grep -v "|" | grep -qw "${database}"; then```

Reply:
Ohhh, the _quotes_!